docs/classad-attributes/scheduler-classad-attributes.rst

```diff
@@ -1155,10 +833,7 @@ These attributes are only reported once a full time span has accumulated.
     name is actually the file transfer queue name, as defined by
     configuration variable ``TRANSFER_QUEUE_USER_EXPR``
 
-:index:`TRANSFER_QUEUE_USER_EXPR`
```

Comment:
I think you've been moving these index lines to before the ad attribute rather than just deleting them?